### PR TITLE
LPS-189054 Add FDS EmptyState sample

### PR DESCRIPTION
**Pull request by edalgrin**
This PR adds FrontendDataSet component with no items in order to trigger the customized EmptyState view

image

### CODE REVIEW

File: modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/SampleEmptyStateFrontendDataSet.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+					},
+				],
+			}}
+			emptyState={{
+				description: Liferay.Language.get(
+					'start-creating-one-to-show-your-data'
+				),
+				image: '/states/empty_state.gif',
+				title: Liferay.Language.get('no-data-sets-created'),
+			}}
```
Comment (markocikos) [inline modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/SampleEmptyStateFrontendDataSet.js:28]: The main goal of this task is to add empty state to tags API. Adding to a sample is a way to immediately check that new API. We don't need a sample of React API for empty state, we already have that tested in DSM.

If you wish, you can split this into two subtasks, first being just to create a placeholder tab in sample module. But, in that case, it should not be a 'fork' of React version of FDS. It should be a fork of Minimum FDS, something like:
```jsp
<frontend-data-set:headless-display
	apiURL="<%= emptyFDSDisplayContext.getAPIURL() %>"
    emptyState='<%=
		HashMapBuilder.<String, String>put(
			"description", "foo"
		).put(
			"image", "bar"
		).put(
			"title", "baz"
		).build()
	%>'
	id="<%= FDSSampleFDSNames.EMPTY %>"
/>
```

Notes on above:
- You can get URL to return an empty set by appending some impossible filter, like `?filter=('color' eq 'foo')`
- We are showing an empty FDS, so we can brand everything `Empty` (tab, FDS name, display context etc). There is no need to call everything `EmptyState`

**Review by markocikos — CHANGES_REQUESTED** [review]
@edalgrin Please see comment inline

File: modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/SampleEmptyStateFrontendDataSet.js (hunk 1 above)
Comment (edalgrin) [reply modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/SampleEmptyStateFrontendDataSet.js:28]: I' going to try/take-a-look, thanks!

Comment (edalgrin) [reply modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/SampleEmptyStateFrontendDataSet.js:28]: I was mislead by the title of the ticket https://liferay.atlassian.net/browse/LPS-189054

**Comment by edalgrin** [comment]
Retry with changes in https://github.com/liferay-frontend/liferay-portal/pull/3575

**Pull request closed without merging**